src/main/java/com/passbook/sparkeighteen/controller/UserController.java

```diff
@@ -38,4 +40,10 @@ public ResponseEntity<LoginResponse> login(@Valid @RequestBody final LoginReques
         return new ResponseEntity<>(userService.login(request), HttpStatus.OK);
     }
 
+    @ApiOperation("delete user profile")
+    @DeleteMapping("/user/{userID}/profile")
+    public ResponseEntity<String> deleteProfile(@PathVariable final Integer userId) {
```

Comment:
What do you mean by delete profile 
Do you also want to delete the user too?

Reply:
yes, profile and user both has one to one relation also user should have only one passbook profile. if i am going to delete that profile user should delete, user without profile shouldn't exist.

Reply:
So will be delete profile or user ?

Reply:
user will delete along with its profile.